src/openttd.cpp

```diff
@@ -947,6 +950,10 @@ static void MakeNewGameDone()
 
 	CheckEngines();
 	CheckIndustries();
+
+	/* Check if the first AI is set to start immediately */
```

Comment:
This is done after creating the human company which takes the first AI line in openttd.cfg. The AI that is started here is the second in openttd.cfg. I don't know if it's fine to say "the first AI".

Reply:
huh? What does openttd.cfg have to do with any of this? Sometimes you talk in riddles.

Reply:
```
[ai_players]
none = start_date=0
"LuDiAI AfterFix" = air_support=1,cities_only=0,is_friendly=1,pf_profile=0,pick_mode=3,road_cap_mode=2,road_min_dist=130,road_support=1,scp_support=0,select_town_cargo=0,start_date=0,station_spread=0
RailwAI = plant_trees=10,start_date=0,use_trams=0
CivilAI = MaxBus=500,MaxCar=0,MinAirRange=256,MinPop=1500,NetworkRadius=4096,ShipRange=128,TrainRange=256,start_date=0
"LuDiAI AfterFix" = is_friendly=1,pf_profile=0,pick_mode=0,road_cap_mode=1,start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
SimpleAI = start_date=0
```
Talking about this.
Maybe first available AI would be better.

Reply:
You know what? Nevermind.